volim.html

```diff
@@ -100,5 +100,21 @@
 			<p class="openhour-lesson__dance">Tab Dancing : 19:00 - 23.00</p>
 		</div>
 	</div>
+  <div class="shortcode">
+   <div class="shortcode-top">
+      <div class="shortcode-title">
+        <p class="shortcode-title__text">SHORTCODE</p>
+        <div class="shortcode-title__circle"></div>
```

Comment:
reuse header title all the time, also please have a look on `shortcode-title__text::after` + content instead of another div with circle. Make it more comple.